config/config.go

```diff
@@ -387,3 +391,12 @@ func PatchStateFile() string {
 
 	return osPatchStateFileLinux
 }
+
+// RestartFile is the location of the restart required file.
```

Comment:
vanity function, move this check into the body code

Reply:
Not sure what you mean by this?

Reply:
this entire function that is switching between two constants based on platform. it's used in one place, and so it essentially replaces an if check with a function call. just do this if check there and remove this function

Reply:
Store store these constants outside of the config package, or make two constants public? Making both of them public expands the public surface of this package making it more complex.

Reply:
Spoke offline, configuration/constant management is probably something we can improve later on in the agent.